alcs-frontend/src/app/features/application/post-decision/edit-reconsideration-dialog/edit-reconsideration-dialog.component.html

```diff
@@ -37,6 +37,42 @@ <h2 class="card-title">Edit Reconsideration</h2>
         </mat-button-toggle-group>
       </div>
 
+      <div *ngIf="isOriginatedInPortal" class="toggle-group">
+        <mat-label class="toggle-label">Reconsideration Type *</mat-label>
+        <mat-button-toggle-group
+          id="reviewOutcome"
+          formControlName="type"
+          name="reviewOutcome"
+          (change)="onTypeReconsiderationChange($event.value)"
+        >
+          <mat-button-toggle *ngFor="let type of codes" value="{{ type.code }}">{{ type.label }}</mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+
+      <div *ngIf="isOriginatedInPortal">
```

Comment:
I believe the benefit of doing this after the "hard fork" is that you don't need to make these fields conditional.

Reply:
The ticket requested to have the reconsideration to be applied only to the once that originated in the portal. Do we ignore those apps that were created before the etl?

Reply:
Updated ticket according to requirements change. The OriginatedInPortal is not relevant anymore and was removed